docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+# Use root/example as user/password credentials
```

Comment:
for running MYSQL locally
start: docker-compose up -d
stop: docker-compose down
view adminer at localhost:8080